db/install.xml

```diff
@@ -62,6 +64,8 @@
         <FIELD NAME="realuserid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="real user id when logged-in-as"/>
         <FIELD NAME="errortype" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="The type of error (Statement send failure, Failure to authenticate with LDH, Network error, Malformed statement etc.)"/>
         <FIELD NAME="response" TYPE="text" NOTNULL="false" SEQUENCE="false" COMMENT="The response or response code received from the LDH"/>
+        <FIELD NAME="logstoreid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="logstore_standard_log_id"/>
```

Comment:
Same comments as per line 30 above

Reply:
I have removed the underscores in the comment and changed this field to logstorestandardlogid and I also use that fieldname in upgrade.php